tutorials/docker/startup.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+Xvfb :0 -screen 0 800x600x24+32 &
```

Comment:
Please add a copyright notice to this file and explain what each of the lines is doing.